src/AaveV3StrategicAssets_20230606/VlTokenManager.sol

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.19;
+
+import {IERC20} from 'solidity-utils/contracts/oz-common/interfaces/IERC20.sol';
+
+import {IVlToken, LockedBalance} from './interfaces/IVlToken.sol';
+import {Common} from './Common.sol';
+
+abstract contract VlTokenManager is Common {
+  event ClaimVLAURARewards();
+  event DelegatedVLAURA(address newDelegate);
+  event EmergencyWithdraw(uint256 tokensUnlocked);
+  event LockVLAURA(uint256 cummulativeTokensLocked, uint256 lockHorizon);
+  event RelockVLAURA(uint256 cumulativeTokensLocked);
+  event UnlockVLAURA(uint256 tokensUnlocked);
+
+  address public constant AURA = 0xC0c293ce456fF0ED870ADd98a0828Dd4d2903DBF;
+  address public constant VL_AURA = 0x3Fa73f1E5d8A792C80F426fc8F84FBF7Ce9bBCAC;
+
+  function lockVLAURA(uint256 amount) external onlyOwnerOrGuardian {
```

Comment:
missing natspec on all the functions in this contract

Reply:
updated